sched/scheduler/stateful_scheduler_test.go

```diff
@@ -215,6 +264,7 @@ func Test_StatefulScheduler_JobRunsToCompletion(t *testing.T) {
 
 	// add additional saga data
 	sagaLogMock.EXPECT().LogMessage(saga.MakeStartTaskMessage(jobId, taskId, nil))
+	sagaLogMock.EXPECT().LogMessage(TaskMessageMatcher{Type: &sagaStartTask, JobId: "job1", TaskId: "task1", Data: gomock.Any()}).AnyTimes()
```

Comment:
It's a race condition whether it sees the process while it's running or if the first poll sees it after it's done, so it's 0 or 1. I've changed it to MaxTimes(1)